app/webserver/helpers.go

```diff
@@ -168,12 +172,15 @@ func createGroup(newGroupData CreateGroupMessage) *store.Session {
 		Members: members,
 	}
 	store.SaveGroup(store.Groups[group.Hexid])
-	session := store.SessionsModel.GetByE164(group.Hexid)
+	if err != nil {
```

Comment:
How `err` can be != nil here, the only place I see err initialized is on the first line of the function and then there is an `if err != nil` just after followed by a `return`, so it looks like this can't happen here.

Reply:
added the missing error initalization